### PR TITLE
Updated Inline PHP Documentation

### LINKED ISSUE #867: Update Inline Documentation
Updated some inline documentation according to  [ PHP Documentation Standards ](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/)

1. **admin/load.php :** Added Global Documentation After since Instead of before since 
2. **modules/database/audit-autoloaded-options/helper.php :** Added  Global Documentation
3. **modules/images/webp-uploads/helper.php :** Added  Global Documentation
4. **modules/js-and-css/audit-enqueued-assets/hooks.php :** Added  Global Documentation

### PR DESCRIPTION
**Pull request by shail-mehta**
Updated some inline documentation according to  [ PHP Documentation Standards ](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/)

Closes https://github.com/WordPress/performance/issues/867

### CODE REVIEW

File: admin/load.php [hunk 1]
```diff
@@ -41,9 +41,9 @@ function perflab_add_modules_page() {
 /**
  * Initializes settings sections and fields for the modules page.
  *
- * @global array $wp_settings_sections Registered WordPress settings sections.
- *
  * @since 1.0.0
+ *
```
Comment (mukeshpanchal27) [inline admin/load.php]:
```suggestion
 *
```

Remove extra space.

File: modules/database/audit-autoloaded-options/helper.php [hunk 2]
```diff
@@ -95,6 +95,8 @@ function perflab_aao_autoloaded_options_test() {
  *
  * @since 1.0.0
  *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
```
Comment (mukeshpanchal27) [inline modules/database/audit-autoloaded-options/helper.php]:
```suggestion
 *
```

File: modules/database/audit-autoloaded-options/helper.php [hunk 3]
```diff
@@ -107,6 +109,8 @@ function perflab_aao_autoloaded_options_size() {
  *
  * @since 1.5.0
  *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
```
Comment (mukeshpanchal27) [inline modules/database/audit-autoloaded-options/helper.php]:
```suggestion
 *
```

File: modules/images/webp-uploads/helper.php [hunk 4]
```diff
@@ -288,6 +288,8 @@ function webp_uploads_get_content_image_mimes( $attachment_id, $context ) {
  *
  * @since 1.3.0
  *
+ * @global WP_Query $wp_query WordPress Query object.
+ *
```
Comment (mukeshpanchal27) [inline modules/images/webp-uploads/helper.php]:
```suggestion
 *
```

File: modules/js-and-css/audit-enqueued-assets/hooks.php [hunk 5]
```diff
@@ -16,6 +16,8 @@
  * It will save information in a transient for 12 hours.
  *
  * @since 1.0.0
+ *
```
Comment (mukeshpanchal27) [inline modules/js-and-css/audit-enqueued-assets/hooks.php]:
```suggestion
 *
```

File: modules/js-and-css/audit-enqueued-assets/hooks.php [hunk 6]
```diff
@@ -59,6 +61,8 @@ function perflab_aea_audit_enqueued_scripts() {
...
```
Comment (mukeshpanchal27) [inline modules/js-and-css/audit-enqueued-assets/hooks.php]:
```suggestion
 *
```

File: modules/js-and-css/audit-enqueued-assets/hooks.php [hunk 7]
```diff
@@ -59,6 +61,8 @@ function perflab_aea_audit_enqueued_scripts() {
  * It will save information in a transient for 12 hours.
  *
  * @since 1.0.0
+ * 
+ * @global WP_Styles  $wp_styles  The WP_Styles current instance.
```
Comment (mukeshpanchal27) [inline modules/js-and-css/audit-enqueued-assets/hooks.php]:
```suggestion
 * @global WP_Styles $wp_styles The WP_Styles current instance.
```

**Review by mukeshpanchal27 — DISMISSED** [review]
Thank you for the pull request. It looks good to me. I've left some feedback, and once those changes are addressed, it will be ready for review.

**Review by swissspidy — APPROVED** [review]

**Comment by shail-mehta** [comment]
Hi @mukeshpanchal27  

Thank you for the feedback. 
I have Updated all changes as per mentioned.

**Review by felixarntz — APPROVED** [review]
Thank you @shail-mehta!

**Pull request merged**